%YAML:1.0

#common parameters
savePath: "/media/seungwon/DataSSD/BAG/deepexpress/final_KAIST/test"
bag_path: "/media/seungwon/DataSSD/BAG/deepexpress/final_KAIST/test/2021-11-22-21-59-25.bag"

scan_topic_name: "/scan"
img_topic_name: "/rgb/image_raw"

# tag info
tag_type: 1 # 1: kalibr_tag, 2: apriltag
tag_size: 0.088 # tag size, unit meter
tag_spacing: 0.3 # tag spacing, only used in kalibr_tag. For details, please see kalibr tag description.
black_border: 2 # if you use kalibr_tag black_boarder = 2; if you use apriltag black_boarder = 1
 
#camera calibration 
model_type: PINHOLE
camera_name: camera
image_width: 1280
image_height: 720

distortion_parameters:
   k1: 0.07962818 
   k2: -0.03559783 
   p1: 0.0010554 
   p2: -0.00025926

projection_parameters:
   fx: 609.31798288 
   fy: 611.7802902  
   cx: 639.26997595 
   cy: 376.02333042

